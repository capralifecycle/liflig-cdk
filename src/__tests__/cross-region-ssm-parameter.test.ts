import { App, Stack } from "aws-cdk-lib"
import "jest-cdk-snapshot"

test("cross-region-ssm-parameter", () => {
  const app = new App()
  const stack1 = new Stack(app, "Stack1", {
    env: {
      region: "us-east-1",
    },
  })

  expect(stack1).toMatchCdkSnapshot()
})
