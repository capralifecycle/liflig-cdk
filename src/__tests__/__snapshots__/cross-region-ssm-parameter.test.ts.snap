// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cross-region-ssm-parameter 1`] = `Object {}`;
